tests/test_plan_stubs.py

```diff
@@ -247,3 +256,132 @@ def test_recover_threshold_timeout_failure(RE, mot_and_sig):
     assert not 49 < pos < 51
     assert mot.position not in (100, -100)
     # If we didn't reach the goal or either end, we timed out
+
+@pytest.mark.timeout(tmo)
+def test_slit_scan_area_compare(RE):
+    fake_slits = Mover(
+        "slits",
+        OrderedDict([
+            ('xwidth',(lambda x,y:x)),
+            ('ywidth',(lambda x,y:y)),
+        ]),
+        {'x':0,'y':0}
+    )
+
+    fake_yag = Reader(
+        'fake_yag',
+        {
+            'xwidth':(lambda:fake_slits.read()['xwidth']['value']),
+            'ywidth':(lambda:fake_slits.read()['ywidth']['value']),
+        }
+    )
+
+    # collector callbacks aggregate data from 'yield from' in the given lists
+    xwidths = []
+    ywidths = []
+    measuredxwidths = collector("xwidth", xwidths)
+    measuredywidths = collector("ywidth", ywidths)
+
+    #test two basic positions
+    RE(
+        run_wrapper(slit_scan_area_comp(fake_slits,fake_yag,1.0,1.0,2)),
+        subs={'event':[measuredxwidths,measuredywidths]}
+    )
+    RE(
+        run_wrapper(slit_scan_area_comp(fake_slits,fake_yag,1.1,1.5,2)),
+        subs={'event':[measuredxwidths,measuredywidths]}
+    )
+    # excpect error if both measurements <= 0
+    with pytest.raises(ValueError):
+        RE(
+            run_wrapper(slit_scan_area_comp(fake_slits,fake_yag,0.0,0.0,2)),
+            subs={'event':[measuredxwidths,measuredywidths]}
+        )
+    # expect error if one measurement <= 0 
+    with pytest.raises(ValueError):
+        RE(
+            run_wrapper(slit_scan_area_comp(fake_slits,fake_yag,1.1,0.0,2)),
+            subs={'event':[measuredxwidths,measuredywidths]}
+        )
+
+    # logger has bad interactin with pytest? only prints on test failure
+    #logger.debug(xwidths) 
+    #logger.debug(ywidths) 
+    print(xwidths) 
```

Comment:
You can just use `logger.debug`. The `run_tests.py` file runs will only show failed messages by default, but the `-s` option will overrule this. Good job leaving a comment!